### PR TITLE
feat: updates for ecotone

### PR DESCRIPTION
**Pull request by kangsorang**
# Description

Updates for ecotone upgrade.

- applying ecotone to devnet
- add ecotone test (multicall3)
- update geth dependency

## Summary by CodeRabbit

- **New Features**
  - Integrated Kroma network support, enhancing blockchain compatibility.
  - Introduced a new contract testing function for the Kroma network.

- **Configuration Updates**
  - Adjusted network initialization settings to improve system configuration.
  - Updated docker configurations to optimize data availability and processing times.

### CODE REVIEW

**Review by 0xHansLee — APPROVED** [review]

**Review by seolaoh — APPROVED** [review]
LGTM

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -49,6 +49,8 @@ jobs:
 
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly-f625d0fa7c51e65b4bf1e8f7931cd1c6e2e285e9
```
Comment (seolaoh) [inline .github/workflows/test.yml:53]: Please apply this to `ci.yml` also.

File: op-chain-ops/cmd/check-ecotone/main.go [hunk 2]
```diff
@@ -372,6 +375,26 @@ func check4788Contract(ctx context.Context, env *actionEnv) error {
...
+	multicall3CodeHash := crypto.Keccak256Hash(multicall3Code)
+
+	multicall3Contract, err := env.l2.CodeAt(ctx, multicall3Address, nil)
+	if err != nil {
+		return fmt.Errorf("failed to retrieve multicall3 contract code: %w", err)
+	}
+	codeHash := crypto.Keccak256Hash(multicall3Contract)
+	if codeHash != multicall3CodeHash {
+		return fmt.Errorf("unexpected multiucall3 contract code: %w", err)
```
[the remark this answers is not in the document]
Comment (Pangssu) [reply op-chain-ops/cmd/check-ecotone/main.go:390]: @kangsorang

**Review by Pangssu — APPROVED** [review]
Check the coderabbit's suggestion

File: op-chain-ops/cmd/check-ecotone/main.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (0xHansLee) [reply op-chain-ops/cmd/check-ecotone/main.go:390]: I think it's ok to use %w for representing error.

**Pull request merged**